### DISCUSSION TITLE
Hold Size keyboard shortcut to resize instead of pressing multiple times

### DISCUSSION THREAD

**Discussion by djpadron** [rxhanson/RectanglePro-Community#103] [Ideas]
Hello all,

I like using the Size > Larger and Size > Smaller keyboard shortcuts to resize my windows.

![image](https://user-images.githubusercontent.com/102387160/189470564-e139c86e-5762-4ce0-bb92-29d3c5b7e7ee.png)

Currently, it seems you have to repeatedly press the shortcut to resize the window.  Is there a setting I can set so that I can **hold** the shortcuts to continuously resize the window?

**Comment by rxhanson** [comment]
Right now the keyboard shortcut recorder only listens to keyup, so I'd have to either modify the shortcut recorder to listen for keydown as well, or get a new shortcut recorder. Both of these are a decent amount of work, which is why I haven't done it yet. I would like to swap out for a new recorder sometime soon, and when I do I'll add this functionality.

**Reply by djpadron** [reply]
Thanks for the reply.
